app/routes/rental_route.py

```diff
@@ -0,0 +1,85 @@
+from app import db
+from app.models.rental import Rental
+from app.models.customer import Customer
+from app.models.video import Video
+from app.routes.customer_route import validate_model
+from flask import Blueprint, jsonify, abort, make_response, request 
+from datetime import datetime
+
+rentals_bp = Blueprint("rentals_bp",__name__, url_prefix="/rentals")
+
+#============================== rentals_bp.route =============================
+#============================================================================
+
+## `POST /rentals/check-out`
+@rentals_bp.route("/check-out", methods=["POST"])
+def create_rental():
+    request_body = request.get_json()
+    try:
+        new_rental = Rental.from_dict(request_body)
+    except:
+        abort(make_response({"message":f"Missing data"}, 400))
+
+    customer = validate_model(Customer, request_body["customer_id"])
+    video = validate_model(Video, request_body["video_id"])
+    
+    if video.total_inventory == 0:
+        abort(make_response({"message":f"Could not perform checkout"}, 400))
+
+    videos_checked_out = Rental.query.filter_by(video_id=video.id).count()  # look into count method
+
+    available_inventory = video.total_inventory - videos_checked_out
+    if available_inventory == 0:
+        abort(make_response({"message":f"Could not perform checkout"}, 400))
```

Comment:
When I see repeated code, I think about added cost of maintenance. In this case, I would wonder if there's a way to extract repeated code.  Maybe at least defining the error string as a variable, so you only have to make changes in one location. :)